### PR TITLE
Updated exclusions from list of exclusions here:

### PR DESCRIPTION
**Pull request by tajhlande**
https://gerrit.wikimedia.org/r/plugins/gitiles/analytics/gobblin/+/refs/heads/wmf/gobblin-modules/gobblin-wmf/build.gradle#72

### CODE REVIEW

File: gobblin-wmf-core/pom.xml [hunk 1]
```diff
@@ -78,5 +78,10 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (gehel) [inline gobblin-wmf-core/pom.xml:81]: Good catch on the missing direct dependency. Looks like we have a few more (`./mvnw dependency:analyze`). The mockito version is missing and needs to be defined in the parent pom.

File: pom.xml [hunk 2]
```diff
@@ -166,6 +226,7 @@
                 <groupId>org.projectlombok</groupId>
                 <artifactId>lombok</artifactId>
                 <version>1.18.18</version>
+                <scope>provided</scope>
```
Comment (gehel) [inline pom.xml:229]: nit: I think the usual pattern for compile-only dependencies is `scope=provided` and `optional=true`. Note that the version (and the scope/optional) are now defined in the parent pom ([discovery-parent-pom](https://github.com/wikimedia/wikimedia-discovery-discovery-parent-pom)) and could be ignored here. To be honest, I'm not sure if it is better to rely on the parent pom or to duplicate the version and scopes.

File: pom.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
     <artifactId>gobblin-wmf</artifactId>
```
Comment (gehel) [inline pom.xml:12]: Now that we cleaned up all those unused dependencies, we might be able to remove some of the exclusions in the configuration of the [duplicate-finder-maven-plugin](https://github.com/basepom/duplicate-finder-maven-plugin) (see below in the <pluginManagement/> section.

**Review by gehel — CHANGES_REQUESTED** [review]
Congratulation on your first PR @WMF!

**Comment by gehel** [comment]
The first commit in this PR is a merge commit from the main branch of the main repo. This looks a bit weird to me. We can always cherry pick the single commit when merging.

File: gobblin-wmf-core/pom.xml (hunk 1 above)
Comment (tajhlande) [reply gobblin-wmf-core/pom.xml:81]: addressed in another commit

File: pom.xml (hunk 3 above)
Comment (tajhlande) [reply pom.xml:12]: Gehel sez .... do this in another PR

**Comment by jobar** [comment]
Thanks a lot for the PR @tajhlande :) This is now merged!

**Pull request merged**